### ISSUE TITLE
Update the styling information on the website

### ISSUE THREAD

**Issue by 00SteinsGate00** [leongersen/noUiSlider#757]
I have worked with noUiSlider in the past before, and now I wanted to use it for another project and I've realized, the styling works differently and that for example the `noUi-background` class was gone.

I looked into the release note and found this

  * Removed: .noUi-stacking class. Handles now stack themselves;
  * Removed: .noUi-handle-lower and .noUi-handle-upper classes;
  * Removed: .noUi-background. This is now default;
  * Removed: connect: 'lower' and connect: 'upper'. These settings are replaced by connect: [true, false];

But the website still mentions those classes

<img width="642" alt="nouislider_screen" src="https://cloud.githubusercontent.com/assets/19982919/22507940/169a2bb6-e888-11e6-96f0-1f0d1545ed56.png">

This should probably be updated to match the new way of styling noUiSlider  (I ended up changing the background through `.noUi-ltr`.

**Comment by leongersen** [comment]
Oops, I should have corrected that. I'll fix it, thanks.

**Comment by leongersen** [comment]
I've updated the documentation.

**Comment by 00SteinsGate00** [comment]
Thank you very much! And thank you for such a great and well maintained piece of software!

One thing I have to note though is, that from this styling documentation I would not know, how to style the background color of the slider bar. I ended up with the `.noUi-ltr` class by looking at the web inspector and trying it with CSS.

Before I just checked the documentation and saw "ok it's `.noUi-background`". I assume that **ltr** means **left-to-right**, so this class is probably not working for vertical and inverted sliders.
